com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added a "Conservative" mode for shader graph depth offset.
 - Added an error message when trying to use disk lights with realtime GI (case 1317808).
 - Added support for multi volumetric cloud shadows.
+- Added an LTC Fitting tools for all the BRDFs supported by HDRP.
+- Adding the support for area lights for Hair and Fabric.
+- using the proper lighting models for AxF for area lights.
 
```

Comment:
- Added LTC Fitting tools for all BRDFs that HDRP supports.
- Added Area Light support for Hair and Fabric master nodes.
- Improved lighting models for AxF shader area lights.